glotaran/builtin/models/kinetic_image/k_matrix.py

```diff
@@ -48,13 +48,14 @@ def involved_compartments(self) -> list[str]:
 
     def combine(self, k_matrix: KMatrix) -> KMatrix:
         """Creates a combined matrix.
+        When combining k-matrices a and b (a.combine(b)),
+        entries in a will overwrite by corresponding entries in b.
```

Comment:
```suggestion

        When combining k-matrices a and b (a.combine(b)),
        entries in a will overwrite by corresponding entries in b.
```
Sorry for being nitpicky, but this would be one less error to fix when we at some point are able to add pydocstyle.